Automattic/wp-calypso
# Onboarding: Picking starting point

#### Changes proposed in this Pull Request

* Implement the picking starting point step and handle the final destination and default theme according to the user's starting point.
* Unify the icons into the icons folder
* Add `SelectItems` component and we can share it between the intent step and picking starting point step.

![image](https://user-images.githubusercontent.com/13596067/137299491-505b6cfc-1b63-427e-9ff3-125bfe8df9e3.png)

![image](https://user-images.githubusercontent.com/13596067/137462116-fa8a4df1-6363-4839-884b-cc041cf49fa0.png)

#### Testing instructions



* Go to `/start/setup-site?flags=signup/hero-flow&siteSlug=<your_site>`
* Select write intent for intent step
* Select Continue for site title and tagline step
* Verify following things
  * Click “Skip to My Home”, you should be redirected to My Home and your theme should be `zoologist`
  * Click “Draft your first post“, you should be redirected to the Editor without “Draft your first post modal” and your theme should be `zoologist`
  * Click “Choose design”. After you select the design, you should be redirected to the Editor with “Draft your first post modal”. This is still waiting https://github.com/Automattic/wp-calypso/pull/56977



Related to https://github.com/Automattic/wp-calypso/issues/56568
